.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           command: npx cypress install
       - run:
           name: Run the node app.
-          command: npm run start-feature --env=cypress.env
+          command: npm run start-feature
```

Comment:
we used to have groovy tests too, but since only cypress tests then don't need the `env` flag